### PR TITLE
ENH: Raw.plot() "zen mode"

### PR DESCRIPTION
**Pull request by drammock**
This adds a "zen mode" for `raw.plot()` (also works with `epochs.plot()` and `ica.plot_sources()`)

- enabled via `raw.plot(show_scrollbars=False)`
- can be toggled by pressing `z` (for "zen mode") when plot has focus even if it wasn't enabled when plot was initialized
- compatible with butterfly mode (if in butterfly mode, vert scrollbar remains hidden when toggling zen mode)
- interaction with subplots_adjust is so/so, but acceptable (assumption is that user won't want/need to use both on one plot: zen mode is for interactive browsing, `subplots_adjust()` compatibility is primarily for making static figures)

cc @cbrnr 

Here's a test script. Try pushing `z` on each test plot, resizing window, pushing `z` before/after resizing again, etc.:

```python
import os
import mne

sample_data_folder = mne.datasets.sample.data_path()
sample_data_raw_file = os.path.join(sample_data_folder, 'MEG', 'sample',
                                    'sample_audvis_raw.fif')
raw = mne.io.read_raw_fif(sample_data_raw_file)
raw.crop(0, 60).load_data()
events = mne.find_events(raw, stim_channel='STI 014')
fig = raw.plot(events=events, show_scrollbars=False)


# EPOCHS
event_dict = {'auditory/left': 1}
reject_criteria = dict(mag=4000e-15,     # 4000 fT
                       grad=4000e-13,    # 4000 fT/cm
                       eeg=150e-6,       # 150 μV
                       eog=250e-6)       # 250 μV
epochs = mne.Epochs(raw, events, event_id=event_dict, tmin=-0.2, tmax=0.5,
                    reject=reject_criteria, preload=True)
epochs.plot(show_scrollbars=False)


# ICA
ica = mne.preprocessing.ICA(n_components=20, random_state=97, max_iter=800)
ica.fit(raw)
ica.plot_sources(raw, show_scrollbars=False)
ica.plot_sources(epochs, show_scrollbars=False)
```

### CODE REVIEW

File: mne/utils/docs.py [hunk 1]
```diff
@@ -282,6 +282,15 @@
     field power).
 """
 
+docdict['zen_mode'] = """
```
Comment (larsoner) [inline mne/utils/docs.py]: Might as well call this show_scrollbars

**Comment by cbrnr** [comment]
(I guess the Azure failure is unrelated?)

**Comment by agramfort** [comment]
rerunning azure now.

+1 for merge when CIs are green.

great work @drammock !

**Comment by cbrnr** [comment]
Thanks @drammock!

**Pull request merged**